app/bundles/CoreBundle/Config/config.php

```diff
@@ -85,13 +85,17 @@
                 'controller' => 'MauticCoreBundle:Api\FileApi:delete',
                 'method'     => 'DELETE',
             ],
-           'mautic_core_api_stats' => [
+            'mautic_core_api_stats' => [
                'path'       => '/stats/{table}',
                'controller' => 'MauticCoreBundle:Api\StatsApi:list',
                'defaults'   => [
                     'table' => '',
                 ],
            ],
+           'mautic_core_api_version' => [
```

Comment:
Do we need an endpoint for this? Can we just include it as a header in the API response here? https://github.com/mautic/mautic/blob/a18a3c8ae868203fdbaddced5fcf6cdb87a970c3/app/bundles/ApiBundle/Controller/CommonApiController.php#L738